test/integration/policy_certificate_test.go

```diff
@@ -32,11 +32,16 @@ var _ = Describe(
 	func() {
 		const (
 			policyCertificateName   = "policy-certificate"
-			policyCertificateURL    = policyCollectSCURL + policyCertificateName + ".yaml"
 			expiredCertSecretName   = "expired-cert"
 			policyCertificateNSName = "policy-certificate"
 		)
 
+		var policyCertificateURL string
+
+		BeforeAll(func() {
+			policyCertificateURL = policyCollectSCURL + policyCertificateName + ".yaml"
+		})
```

Comment:
This feels a little odd. Why not just move the variable declarations to the `init()` function?

Reply:
Are the flags parsed by then?

Reply:
I just double checked and the flags are not parsed by then which is why I opted for BeforeSuite but BeforeSuite doesn't run until after the tests are detected by Ginkgo which leads to the BeforeAll.

Reply:
@dhaiducek I think it used to be an issue but now it seems I can call `flag.Parse` outside of Ginkgo.

Reply:
Seems the "proper" way is to put it in `TestIntegration()`? https://onsi.github.io/ginkgo/#overriding-ginkgos-command-line-configuration-in-the-suite

Does that work? It looks like `go test` might also call `flag.Parse()`, so I'm not sure of the effect of it being called twice...

Reply:
Good find. It did seem to be okay with `flag.Parse()` being called twice but I'll try what you suggested.